### ISSUE TITLE
Support Lua runtime files

### ISSUE THREAD

**Issue by wbthomason** [wbthomason/packer.nvim#403]
We need to add support for explicitly sourcing Lua runtime files of newly installed plugins, now that https://github.com/neovim/neovim/pull/14686 has been merged.

**Comment by shadmansaleh** [comment]
Do we ? pack plugins are handled in core :D 

I can think of small things like running ftdetect/*.lua for opt plugins to lazyload on `ft` config. But other then that I think we are fine already .

**Comment by wbthomason** [comment]
I was thinking of `ftdetect` but also the case handled here: https://github.com/wbthomason/packer.nvim/blob/master/lua/packer/plugin_utils.lua#L148-L170

**Comment by shadmansaleh** [comment]
I'm not sure why it's there ... I though pack would add the   start plugins to rtp & exexute the `plugin/**/*.(vim|lua)` . Also I think `after/plugin/**/*.vim` aren't suppose to be executed :/

**Comment by wbthomason** [comment]
Yes - that function is for (1) loading newly installed plugins without a restart and (2) ensuring plugins are loaded before running post-update hooks if those hooks use commands defined in the plugin.
